main.go

```diff
@@ -50,10 +58,18 @@ func init() {
 
 func main() {
 	var metricsAddr string
+	var leaderElectionNamespace string
 	var enableLeaderElection bool
 	var probeAddr string
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
-	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+
+	cfg := config.NewDriverConfig()
+
+	flag.StringVar(&metricsAddr, "metrics-bind-address", "0", "The address the metric endpoint binds to.")
+	flag.StringVar(&leaderElectionNamespace, "leader-election-namespace", "", "Namespace where the leader election resource lives")
+	flag.StringVar(&cfg.DriverName, "driver-name", "", "The CSI driver name.")
+	flag.StringVar(&cfg.DriverEndPoint, "csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
+	flag.DurationVar(&cfg.Timeout, "timeout", defaultTimeout, "The timeout for RPCs to the CSI driver.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":9998", "The address the probe endpoint binds to.")
```

Comment:
Should we keep this as before and change it in our deployment configuration? Or do you want to enforce this default to everyone (unless they want to change it) ?

Reply:
changed it 9998 as 8081 is the most common and widely used one.

Reply:
I had some other thoughts, but I this is fine too.